test/Mvc6Framework45.FunctionalTests/project.json

```diff
@@ -1,9 +1,9 @@
 {
-  "webroot": "wwwroot",
-  "userSecretsId": "aspnet5-Mvc6Framework45.FunctionalTests-42f45116-363a-40d3-9c44-bc58885d6e96",
-  "version": "1.0.0-*",
+    "webroot": "wwwroot",
```

Comment:
Whitespace differences make reviewing large pull requests like this more difficult. :cry: